R/read.any.R

```diff
@@ -64,7 +66,7 @@ read.any <- function(
     file_type = '', 
     
     first_column_name = NA, 
-    header = TRUE, 
+    header = is.null(col.names), 
```

Comment:
Please add a comment here for future devs, something like "# col.names indicates fixed-width files which typically do not have hearders."

Also, I just realized this might break if a user reads FWF with file_type = 'FWF' and doesn't give col.names, but they'll have to give widths, so `is.null(widths)` is better, please use that instead.